plugins/inputs/tail/README.md

```diff
@@ -67,6 +67,9 @@ The plugin expects messages in one of the
   ## Set the tag that will contain the path of the tailed file. If you don't want this tag, set it to an empty string.
   # path_tag = "path"
 
+  ## Remove ANSI colors (the only supported value now)
+  # remove = ["colors"]
```

Comment:
I like that you're using a string setting now. It gives us flexibility if we want to add another option in the future. Could we could use a more generic term like "filters" or "transformations"? That way we will have the future option to add any other processing step, instead of just "removing". We could add one to make everything lowercase or translate to pig latin or whatever.

Also, by convention, the commented out setting shows the default value. The plugin doesn't default to removing color, so we should change the comment. see https://github.com/influxdata/telegraf/blob/master/docs/developers/SAMPLE_CONFIG.md#style

```suggestion
  ## Filters to apply to files before generating metrics
  ## "ansi_color" removes ANSI colors
  # filters = []
```

Reply:
Thanks for the review @reimda! Updated the PR.

Reply:
Thanks @powersj, done